test/jdk/javax/net/ssl/TLSCommon/interop/BaseInteropTest.java

```diff
@@ -225,6 +224,23 @@ protected Status runTestCase(TestCase<U> testCase) throws Exception {
         return caseStatus;
     }
 
+    /*
+     * Return a server once it is properly started to avoid client connection issues.
+     */
+    protected AbstractServer startAndGetServer(U useCase, ExecutorService executor)
+            throws Exception {
+        AbstractServer server = createServer(useCase, executor);
+        if (!Utilities.waitFor(Server::isAlive, server)) {
+            // Retry operation, server might have failed to bind a port
+            server.signalStop();
+            server = createServer(useCase, executor);
```

Comment:
Can you please define MAX_RETRIES and use loop to retry? This will make it flexible for us to update retries if needed and avoid duplicate code.

Reply:
Done, also retries value can be now overridden by subclasses if we wanted. In addition, moved Utilities::shell and shellProc to ProcUtils class which is a better location and added support for running win commands